eservice/pdo/eservice/enclave/swig_utils.cpp

```diff
@@ -152,9 +160,20 @@ void PyLogV(
     va_start(ap, message);
     vsnprintf_s(msg, BUFFER_SIZE, BUFFER_SIZE-1, message, ap);
```

Comment:
how about switching to the 4-parameter version of vsnprintf_s and getting rid of the 5-parameter one (or the opposite)?
In our implementation (common/c11_support.cpp) the 5p version is equivalent to the 4p one.

FYI: the 4p version has been defined in C11 (https://en.cppreference.com/w/c/io/vfprintf), while the 5p version has been defined by Microsoft.

Reply:
Updated.